demo-app/src/pages/servers.jsx

```diff
@@ -1,20 +1,81 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { Link } from 'react-router-dom';
-import { Page, PageContent, PageHeader } from 'grommet';
-import { ReverseAnchor } from '../components';
+import {
+  Anchor,
+  Box,
+  Data,
+  DataSearch,
+  DataFilters,
+  DataSummary,
+  Grid,
+  Heading,
+  Menu,
+  Page,
+  PageContent,
+  PageHeader,
+  ResponsiveContext,
+  Toolbar,
+} from 'grommet';
+import { LinkNext } from 'grommet-icons';
+import {
+  ContentContainer,
+  ServersMap,
+  DeviceGroups,
+  RecentActivity,
+  ReverseAnchor,
+} from '../components';
+import data from '../data/servers.json';
 
 const Servers = () => {
+  const breakpoint = useContext(ResponsiveContext);
+
   return (
-    <Page fill>
-      <PageContent flex>
+    <Page kind="full" pad={{ bottom: 'xlarge' }}>
+      <PageContent>
         <PageHeader
           title="Servers"
           parent={<ReverseAnchor as={Link} label="Home" to="/" />}
         />
-        placeholder
+        <Grid
+          columns={
+            !['xsmall', 'small', 'medium'].includes(breakpoint)
+              ? ['flex', 'medium']
+              : ['auto']
+          }
+          gap="medium"
+        >
+          <ContentContainer gap="large">
+            <DataView />
+            <Box gap="medium">
```

Comment:
Do we need this Box with gap="medium" since it only has 1 child, which is another Box?

Reply:
Good catch, DeviceGroups should be inside that Box. adjusting.

Reply:
Donne.